.travis.yml

```diff
@@ -24,7 +24,7 @@ script:
 
 jobs:
   allow_failures:
-    - php: 7.4snapshot
+    - php: 7.4
```

Comment:
Making a note that we should remove this section before releasing the next patch release since we're explicitly targeting PHP 7.4 with it.

Reply:
I just removed it now, otherwise we forget anyways and since its not failing anyways its just a minor cleanup.

Reply:
Ah, I was expecting it to fail. If it's passing, we're good 🎉